package.xml

```diff
@@ -0,0 +1,16 @@
+<package format="2">
+  <name>fcl</name>
+  <version>0.6.0</version>
+  <description>FCL: the Flexible Collision Library</description>
+  <maintainer email="jslee02@gmail.com">Jeongseok Lee</maintainer>
```

Comment:
I'm not currently maintaining FCL actively. I guess @sherm1 or @SeanCurtis-TRI would be more make sense to be the maintainer.

Reply:
OK, I changed it to both of them.